docs/source/pages/tutorials/use_verification_summary_attestation.rst

```diff
@@ -0,0 +1,159 @@
+.. Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+============================================
+How to use Verification Summary Attestations
+============================================
+
+This tutorial explains how to use the Verification Summary Attestations (VSA) generated by Macaron, using the VSAs for the `Graal Development Kit (GDK) <https://graal.cloud/gdk/>`_ artifacts as an example.
+
+For more information about VSAs, please refer to the :ref:`Verification Summary Attestation page<vsa>`. To use Macaron to generate VSAs see this :ref:`tutorial <gen-vsa_tutorial>`.
+
+* https://slsa.dev/spec/v1.0/verification_summary
+
+--------
+Use case
+--------
+
+Imagine you are a consumer of GDK artifacts and want to verify whether they are produced by a secure, :term:`SLSA`-compliant build service and sourced from a trusted source code repository. You need to confirm whether a provenance record for this artifact is published and has been verified. The GDK team must keep the details of their build pipeline confidential while still communicating that verification has occurred.
+
+A VSA allows you to assess the security properties of an artifact without needing direct access to the provenance details. This process involves delegating the policy decision to Macaron. Macaron receives the provenance of the build as input, analyzes various aspects of the build, and verifies the gathered data against a Datalog policy. It then generates a VSA that attests to the artifacts produced by the build, which is published alongside the artifacts.
```

Comment:
```suggestion
A VSA allows you to assess the security properties of an artifact without needing direct access to the provenance details. This process involves delegating the policy decision to Macaron. Macaron receives the provenance of the build as input, analyzes various aspects of the build, and verifies the gathered data against a Datalog policy. It then generates a VSA, which is published alongside the artifacts, that attests to the artifacts produced by the build.
```
I think we can prevent confusion on what is published alongside the artifacts if we change this sentence into this version.

Reply:
See commit [32e104c](https://github.com/oracle/macaron/pull/858/commits/32e104c12ea49dec6675910dc6e9a3ea59baf4d9)